src/selectors/layers.js

```diff
@@ -13,14 +13,17 @@ export const getLayers = createSelector(
     const bounds = {};
 
     for (const node of nodes) {
-      const bound =
-        bounds[node.layer] || (bounds[node.layer] = [Infinity, -Infinity]);
-      if (node.y - node.height < bound[0]) bound[0] = node.y - node.height;
-      if (node.y + node.height > bound[1]) bound[1] = node.y + node.height;
+      const layer = node.nearestLayer || node.layer;
+
+      if (layer) {
+        const bound = bounds[layer] || (bounds[layer] = [Infinity, -Infinity]);
+        if (node.y - node.height < bound[0]) bound[0] = node.y - node.height;
+        if (node.y + node.height > bound[1]) bound[1] = node.y + node.height;
```

Comment:
I'd prefer these not to be inline if statements and to add braces, just for improved readability? Is that ok with you?

Reply:
Good point I've added these.